pkg/cmd/datamgr/cli/server/server.go

```diff
@@ -232,6 +232,13 @@ func newServer(f client.Factory, config serverConfig, logger *logrus.Logger) (*s
 		return nil, err
 	}
 
+	clusterFlavor, err := utils.GetClusterFlavor(clientConfig)
```

Comment:
Any specific reason about moveing GetClusterFlavor up here?

Reply:
Just to make logging consistent, the backup-driver startup print the cluster flavor in 2nd line, I just wanted the same thing in data-manager too.